xml/System/IntPtr.xml

```diff
@@ -2889,7 +2895,7 @@ This member is an explicit interface member implementation. It can be used only
         <ReturnType>System.IntPtr</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>A read-only field that represents a pointer or handle that has been initialized to zero.</summary>
+        <summary>A read-only field that represents a signed integer that has been initialized to zero.</summary>
```

Comment:
Similarly to `Add`. Users should likely prefer `(nint)0` or `(nuint)0` instead. It is smaller and more efficient IL and is more closely similar to how you represent `Zero` for other integer types.